/bench_tracker_v3:
  ros__parameters:
    Kd_offset: 1.0
    Kp_offset: 0.5
    Kp_offset_b: 0.05
    base_rpm: 12.0
    corr_rpm: 2.0
    d_filter_t: 1.0
    max_rpm: 25.0
    offset_enter_m: 0.1
    offset_exit_m: 0.01
    steer_settle_s: 2
    use_sim_time: false
    yaw_enter_m: 0.05
    yaw_exit_m: 0.01
